usaspending_api/references/models.py

```diff
@@ -50,7 +50,7 @@ class Meta:
 class Agency(models.Model):
     # id = models.AutoField(primary_key=True)
     id = models.AutoField(primary_key=True)  # meaningless id
-    cgac_code = models.CharField(max_length=3, blank=True, null=True, verbose_name="Agency Code")
+    cgac_code = models.CharField(max_length=6, blank=True, null=True, verbose_name="Agency Code")
```

Comment:
@bsweger @afrasier  CGAC codes increased from length of 3 to 6 based on agency feedback and latest version of the IDD.